### PR TITLE
DOCS-8742-sockets-index-kt

### PR DESCRIPTION
**Pull request by kevintroller**
Index. adoc editorial

### CODE REVIEW

File: sockets/1.2/modules/ROOT/pages/edits/index.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+= Sockets Connector 1.2 - Mule 4
+:page-aliases: connectors::sockets/sockets-connector.adoc
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Sockets (Sockets Connector) provides a sockets listener, send capability, and send and receive functionality. Additionally, provides TCP and UDP requesters.
```
Comment (kbostrom) [inline sockets/1.2/modules/ROOT/pages/edits/index.adoc]:
```suggestion
Anypoint Connector for Sockets (Sockets Connector) provides a sockets listener, send capability, and send and receive functionality. Additionally, it provides TCP and UDP requesters.
```

Comment (kbostrom) [reply sockets/1.2/modules/ROOT/pages/edits/index.adoc]: Do we need to say anything about WHY or under what circumstances our audience wants or needs these functionalities? Or is that just too obvious?

File: sockets/1.2/modules/ROOT/pages/edits/index.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+== Audience
+
+* New user, read:
+** xref:sockets-connector-studio.adoc[Use Studio to Configure the Connector] to create your Mule app
+** xref:sockets-connector-examples.adoc[Examples], which provides one or more use cases for the connector
+
+* Advanced user, read:
+** xref:sockets-connector-xml-maven.adoc[XML and Maven Support]
+** xref:sockets-connector-examples.adoc[Examples], which provides one or more use cases for the
```
Comment (kbostrom) [inline sockets/1.2/modules/ROOT/pages/edits/index.adoc]:
```suggestion
** xref:sockets-connector-examples.adoc[Examples], which provides one or more use cases for the connector
```

**Review by kbostrom — DISMISSED** [review]
Edited per scope of request

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**